phpBB/phpbb/event/data.php

```diff
@@ -63,4 +63,17 @@ public function offsetUnset($offset)
 	{
 		unset($this->data[$offset]);
 	}
+
+	/**
+	 * Returns data with updated key in specified offset.
+	 *
+	 * @param	string	$offset	Data array offset
+	 * @param	string	$key	Offset key
+	 * @param	mixed	$value	Value to update
+	 * @return	mixed	Value of $value
```

Comment:
We either return nothing or `$value`. There is nothing more to that, I just followed what other methods in the class do.